homeassistant/components/wevolor/translations/en.json

```diff
@@ -0,0 +1,21 @@
+{
```

Comment:
Don't forget to run the translations script after every change of `strings.json`.
```
python3 -m script.translations develop
```